javascript/javascript1/week2/weather-wear.js

```diff
@@ -0,0 +1,29 @@
+function weatherChange(temperature){
+    if(temperature>=30){
+    return console.log("Too hot,wear t-shirts and shorts");
```

Comment:
The function is supposed to return a string describing what to wear. It is not the functions responsibility to print what clothes to wear in the console log:
```js
...
    if(temperature>=30) {
        return "Too hot,wear t-shirts and shorts";
...
```

Who ever calls the function can decide to print it to the console log, or output it on a website or put it in an automatic email and send it.

Notice how the [homework shows an example](https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript1/week2/homework.md#weather-wear):
```js
const clothesToWear = youCreateThisFunctionName(18);
console.log(clothesToWear); // Logs out: "shorts and a t-shirt"
```

First, call the function and save the return value in the variable `clothesToWear`, then print it using `console.log` (the printing happens OUTSIDE the function).